CONTRIBUTING.md

```diff
@@ -18,6 +18,14 @@ Tip: run `ln -sf $(pwd)/build/* ~/.wakatime/` to have the plugins use your local
 
 ## Branches
 
+PR branch names must use one of the following prefixes:
+
+- `^major/.+` - `major`
+- `^feature/.+` - `minor`
+- `^bugfix/.+` - `patch`
+- `^docs?/.+` - `build`
+- `^misc/.+` - `build`
+
```

Comment:
It's already specified [here](https://github.com/wakatime/wakatime-cli/blob/develop/CONTRIBUTING.md#branching-stratgegy)

Reply:
Yea, but I want it in this doc so we don't have to follow an external link.